app/graphql/mutations/systemnotifications/create_system_notifications.rb

```diff
@@ -0,0 +1,20 @@
+module Mutations
+  class CreateSystemNotification < BaseMutation
+    argument :title, String, required: true
+    argument :content, String, required: true
+
+    field :system_notification, Types::SystemNotificationType, null: true
+    field :errors, [String], null: false
```

Comment:
No need for this, base mutation should be able to handle any error, check that file for more info